src/AdoNet/Shared/Storage/DbExtensions.cs

```diff
@@ -139,6 +139,20 @@ public static TValue GetValueOrDefault<TValue>(this IDataRecord record, string f
             }
         }
 
```

Comment:
/// <summary>
        /// Returns a value if it is not <see cref="System.DBNull"/>, <em>default</em>  otherwise.
        /// </summary>
        /// <param name="record">The record from which to retrieve the value.</param>
        /// <param name="fieldName">The name of the field to retrieve.</param>
        /// <param name="default">The default value if value in position is <see cref="System.DBNull"/>.</param>
        /// <returns>Either the given value or the default for DateTime?.</returns>
        /// <exception cref="DataException"/>
        /// <remarks>An explicit function like this is needed in cases where to connector infers a type that is undesirable.
        /// An example here is Npgsql.NodaTime, which makes Npgsql return Noda type and consequently Orleans is not able to
        /// use it since it expects .NET DateTime. This function throws if the given <see paramref="fieldName"/> does not exist.</remarks>

Reply:
You can propose this as change using the +- icon, @dave-b-code
![image](https://user-images.githubusercontent.com/203839/148103538-ec73c928-05b6-4efd-adc8-48c08cc00785.png)